helm/cluster-openstack/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: cluster-openstack
+description: A Helm chart for Kubernetes
+type: application
+version: "[[ .Version ]]"
```

Comment:
Do we need this? Otherwise let's remove it. I mean the `[[ .Version ]]` architect part.

Reply:
We currently need it as we use architect, we would not need it with ABS AFAIK but the cute `explicit_allow_chart_name_mismatch` does not seem to work with ABS